charts/linkerd2/templates/heartbeat.yaml

```diff
@@ -17,7 +17,12 @@ metadata:
   annotations:
     {{ include "partials.annotations.created-by" . }}
 spec:
+  concurrencyPolicy: Replace
+  {{ if .Values.heartbeatSchedule -}}
   schedule: "{{.Values.heartbeatSchedule}}"
+  {{ else -}}
+  schedule: "{{ dateInZone "04 15 * * *" (now | mustDateModify "+10m") "UTC"}}"
```

Comment:
What does `dateInZone "04 15 * * *"` do here? I understand that to get the first heartbeat, it should be now + 10m (`now | mustDateModify "+10m"`), but why `04 15 * * *` setting? That would be 15:04 everyday, right?

Reply:
That's just go time formatting magic, based on the reference time "Mon Jan 2 15:04:05 -0700 MST 2006"  :-)
So "04 15 * * *" means minutes followed by hours, followed by stars.